packages/fxa-auth-server/lib/senders/emails/layouts/subscription/en.ftl

```diff
@@ -1,3 +1,7 @@
+subplat-header-firefox-logo = <img data-l10n-name="fxa-logo-firefox" alt="Firefox logo">
```

Comment:
Per [fluent documentation](https://projectfluent.org/dom-l10n-documentation/dom-overlays.html), this is the way to include an image, we just wanted to doublecheck the following:
(1) `Firefox logo` would be translated with just this line (line 1)?
(2) If the alt text is translated in line 1, we don't believe line 2 (`.alt = { -brand-firefox } logo`) is necessary and should be removed (as well as line 4). Can you please confirm? Thank you!
cc: @LZoog

Reply:
I'm not sure if the alt text would literally appear as `{ -brand-firefox } logo` if we wrote it like that?